### ISSUE TITLE
[Feature Request] A system tray icon that acts like a mini Gamehub game search

### ISSUE THREAD

**Issue by Toadfield** [tkashkin/GameHub#510]
So what I mean is,a system tray icon,and that if you click it,it doesn't open gamehub,but a mini search thing for all gamehub games.
So if you click on it,you can search for game xyz and if you click on it,then it opens the game with all the tweaks and stuff.
And you should be able to filter like with the normal gamehub and that you can right click the game there and select all the thing that you can do in normal gamehub (like "Run with compatibility layer","Remove","Properties" being able to set that tags etc.).
And if you right click a game that you can select something like "Show in Gamehub" and then it opens the gamehub window and you can view it there.
If that setting will be implemented,then it would be cool if it is disabled by default,but that you can enable it in the gamehub settings.

**Comment by Toadfield** [comment]
I have a better Idea,that it is not a system tray icon,but a widget,that would be cooler.

**Comment by Toadfield** [comment]
A cool addition would be,to be able to set,that it shows your recent and most used games/programs first if you click on that widget without searching anything.

**Comment by Toadfield** [comment]
A cool addition would be to be that there is a little button and if you click that,that it opens gamehub.

**Comment by Lucki** [comment]
GameHub is designed for Elementary OS which uses Gnome 3. Gnome 3 neither have a tray or widgets so keep your expectations low. For the tray there is already a discussion and implementation in #430 

Your general idea is a search feature which in itself is perfectly supported in the Gnome environment by search providers which are listing the results in the global search. https://developer.gnome.org/SearchProvider/

**Comment by Toadfield** [comment]
> Your general idea is a search feature which in itself is perfectly supported in the Gnome environment by search providers which are listing the results in the global search.

Yeah,that would be better,if it is in the normal gnome search/krunner implemented,so with all the features I listed in the post,but instead of a widget,in gnome search/krunner.
